src/minter.ts

```diff
@@ -112,13 +124,23 @@ export class Minter {
       // corrects MIME type.
       const tFile = await correctFileType(file)
 
-      if (tFile.size > FILE_UPLOAD_SIZE_LIMIT)
+      if (
+        tFile.size >
+        (this.constants.FILE_UPLOAD_SIZE_LIMIT || FILE_UPLOAD_SIZE_LIMIT)
+      )
         throw new Error(ERROR_MESSAGES.fileSizeExceeded)
 
-      const result = await uploadToArweave(file, this.apiKey)
+      if (!this.storage) throw new Error('Storage not initialized')
```

Comment:
Should check storage instance initialization before filesize check

Reply:
👍